src/SkyApm.Diagnostics.HttpClient/Config/HttpClientDiagnosticConfig.cs

```diff
@@ -32,7 +32,7 @@ public class HttpClientDiagnosticConfig
         /// <summary>
         /// Request/Response body will skip collecting if the content length is larger than this value.
         /// </summary>
-        public int CollectBodyLengthThreshold { get; set; }
+        public int CollectBodyLengthThreshold { get; set; } = 2048;
```

Comment:
Whether default value makes the agent collects the body always? This kind of gathering data usually has a performance impact.

Reply:
agent will try to collect the body only when the Content-Type matches CollectRequestBodyContentTypes.
the CollectBodyLengthThreshold is intend for skip collect in case the body is too large.

Reply:
agent will try to collect the body only when the Content-Type matches CollectRequestBodyContentTypes.
the CollectBodyLengthThreshold is intend for skip collect in case the body is too large.